### PR TITLE
API Route: Driver Query

### LINKED ISSUE #46: Create API Route for Admin to get driver information
Create API Route for Admin to search and get user information as per filter applied by admin
User info can be filtered by:
1. Name {fname, mname, lname}
1. Gender
1. Age Range
1. Address {Area, City, State, Pincode}
1. earning range
1. Rating
1. license exp date

### PR DESCRIPTION
**Pull request by malivinayak**
Creating API Route for Admin to search and get user information as per filter applied by admin
Driver info can be filtered by:
1. Name {fname, mname, lname}
1. Gender
1. Age Range
1. Address {Area, City, State, Pincode}
1. earning range
1. Rating

Closes #46

### CODE REVIEW

File: src/backend/routes/admin/accessDriver.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            query += ageR1 ? `E.age() >=${Math.floor(ageR1)} and ` : '';
+            query += ageR2 ? `E.age() <=${Math.floor(ageR2)} and ` : '';
+            query += area ? `E.DADDRESS.AREA like '%${area}%' and ` : '';
+            query += city ? `E.DADDRESS.CITY like '%${city}%' and ` : '';
+            query += state ? `E.DADDRESS.STATE like '%${state}%' and ` : '';
+            query += pincode ? `E.DADDRESS.PINCODE like '%${pincode}%' and ` : '';
+            query += earningR1 ? `TOTAL_EARING >=${Math.floor(earningR1)} and ` : '';
+            query += earningR2 ? `TOTAL_EARING <=${Math.floor(earningR2)} and ` : '';
+            query += ratingR1 ? `AVG_RATING >=${ratingR1} and ` : '';
+            query += ratingR2 ? `AVG_RATING <=${ratingR2} and ` : '';
```
Comment (OutdatedGuy) [inline src/backend/routes/admin/accessDriver.js:48]: This syntax would work with oracle??

Comment (malivinayak) [reply src/backend/routes/admin/accessDriver.js:48]: Yes I tested it with postman
I am pushing commits only after testing and retrieving data using postman

Comment (malivinayak) [reply src/backend/routes/admin/accessDriver.js:48]: I have tested for user Query with same syntax
So used as it is for driver

Comment (malivinayak) [reply src/backend/routes/admin/accessDriver.js:48]: there are some issues while executing such queries but I made my way and now it works properly

**Review by OutdatedGuy — APPROVED** [review]

**Pull request merged**